packages/e2e/src/build-parachain-block.test.ts

```diff
@@ -0,0 +1,50 @@
+import { afterAll, describe, expect, it } from 'vitest'
+
+import { TypeRegistry } from '@polkadot/types'
+import { decodeProof } from '@acala-network/chopsticks-core'
+import { setupAll } from './helper.js'
+import { upgradeRestrictionSignal } from '@acala-network/chopsticks-core/utils/proof.js'
+
+describe.runIf(process.env.CI || process.env.RUN_ALL).each([
```

Comment:
there's no need for this many tests
A single chain will do
```suggestion
describe('override-relay-state-proof', async () => {
  const { chain, dev, teardown, api } = await networks.acala()
  ...
```

Reply:
Applied your suggestions